### PR TITLE
Feature/sc 32780/learningobject hierarchy module route updates

### PR DESCRIPTION
**Pull request by Romerico234**
All the routes in this module were good. I renamed some parameters and removed unused modules/parameters.

### CODE REVIEW

File: src/app/core/learning-object-module/hierarchy/hierarchy.service.ts [hunk 1]
```diff
@@ -60,12 +59,11 @@ export class HierarchyService {
   /**
    * Adds children to a learning object
    *
-   * @param username the username of the author
    * @param object the object having children
    * @param children the children to be had
    * @returns
    */
-  async addChildren(username: string, object: any, children): Promise<any> {
```
Comment (joaquinVill) [inline src/app/core/learning-object-module/hierarchy/hierarchy.service.ts:68]: Is username being taken care of somewhere else or do we not need username at all?

**Review by stef359 — APPROVED** [review]

File: src/app/core/learning-object-module/hierarchy/hierarchy.service.ts (hunk 1 above)
Comment (Cwagne17) [reply src/app/core/learning-object-module/hierarchy/hierarchy.service.ts:68]: I don't think we ned username, was it not being referenced anywhere?

File: src/app/core/learning-object-module/hierarchy/hierarchy.service.ts [hunk 2]
```diff
@@ -60,12 +59,11 @@ export class HierarchyService {
...
+  async addChildren(object: any, children): Promise<any> {
```
Comment (Cwagne17) [inline src/app/core/learning-object-module/hierarchy/hierarchy.service.ts:66]: What type is object? Is it LearningObject?

**Review by Cwagne17 — COMMENTED** [review]
Questions.

File: src/app/core/learning-object-module/hierarchy/hierarchy.service.ts (hunk 1 above)
Comment (Romerico234) [reply src/app/core/learning-object-module/hierarchy/hierarchy.service.ts:68]: Username parameter wasn't being referenced anywhere

**Comment by Romerico234** [comment]
Renamed and added types for the addHierarchyObject and addChildren parameters

File: src/app/core/learning-object-module/hierarchy/hierarchy.service.ts (hunk 2 above)
Comment (Romerico234) [reply src/app/core/learning-object-module/hierarchy/hierarchy.service.ts]: Yes, the types for both parameters is LearningObject

**Review by pzalep1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Review by joaquinVill — APPROVED** [review]

**Pull request merged**